lib/home_screen.dart

```diff
@@ -149,7 +165,7 @@ class _HomePageState extends State<HomePage> {
                 padding: EdgeInsets.only(top: 130, left: 38),
                 child: Container(
                   child: Text(
-                    documentData["Username"],
```

Comment:
why is divyansh gupta is being used as universal name instead it should be showing the name of the user whose account is logged in.